Resources/Prototypes/_LostParadise/Entities/Objects/Specific/Cyborgs/borg_modules_syndi_sabotage.yml

```
@@ -0,0 +1,57 @@
+    - SyndicateJawsOfLife
+    - PowerDrill
+
+
+- type: entity
+  name: cybor radio jammer
+  parent: RadioJammer
+  id: LPPRadioJammerCyborg
+  description: Cyborg version of device, which will disrupt any nearby outgoing radio communication as well as suit sensors when activated.
+  slots:
+      cell_slot:
+        name: power-cell-slot-component-slot-name-default
```

yamllint (1.35.1)

[error] wrong indentation: expected 10 but found 8

(indentation)